lib/src/linting/find-lint-violations.ts

```diff
@@ -0,0 +1,73 @@
+import { GroupedLintRuleViolations } from "./rule";
+import { LintConfig } from "cli/src/commands/lint";
+
+interface FindLintViolationsDependencies {
+  error: (
+    msg: string,
+    options: {
```

Comment:
We never set `code` and we never set `exit: true` here. Could we remove `options` altogether, and instead do the following in `commands/lint.ts`:
```
    const { errorCount } = findLintViolations(groupedLintErrors, lintConfig, {
      error: (msg) => this.error(msg, { exit: false }),
      warn: this.warn
    });
```

Reply:
👍 I ended up removing options from the interface. Looking at the oclif readme, it looks like we have to specify `{exit: false }` for it to not exit but we can just pass in:
```
(msg: string) => {
        this.error(msg, { exit: false });
},
```